app/models/course.rb

```diff
@@ -35,6 +35,7 @@ class Course < ActiveRecord::Base
                                 source: :actable, source_type: Course::LessonPlan::Event.name
   has_many :material_folders, class_name: Course::Material::Folder.name, inverse_of: :course,
                               dependent: :destroy
+  has_many :forums, dependent: :destroy
```

Comment:
Thanks, I understand, there still a lot of things to do with the forum and discussion, and I'll make my PR smaller and more clear.